R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -380,7 +392,7 @@ udb_roll_avg_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
 #' @param rev_time ISO-8601-formatted string specifying the look-behind window size.
 #' @param fwd_time ISO-8601-formatted string specifying the look-ahead window size. Default is 0 seconds.
 #' @export
-udb_roll_avg_time <- function(ts_col, cols, rev_time, fwd_time = "PT0s") {
+uby_rolling_avg_time <- function(ts_col, cols, rev_time, fwd_time = "PT0s") {
```

Comment:
If this is a simple moving average, that and SMA should be indicated in the docstring.